docs/extensibility.md

```diff
@@ -29,531 +28,630 @@ pub trait DiscoveryHandler {
+The first step is to create a DiscoveryHandler configuration struct. This struct will be used to deserialize the CRD contents and will be passed on to our HttpDiscoveryHandler. Here we are specifying that users must pass in the URL of a discovery service which will be queried to find our HTTP-based Devices.  Add this code to `shared/src/akri/configuration.rs`:
 
 ```rust
+/// This defines the HTTP data stored in the Configuration
 #[derive(Serialize, Deserialize, Clone, Debug)]
 #[serde(rename_all = "camelCase")]
-pub struct NessieDiscoveryHandlerConfig {
-    pub nessie_url: String,
+pub struct HTTPDiscoveryHandlerConfig {
+    pub discovery_endpoint: String,
 }
 ```
 
-Next, we need to update the Akri protocol handler enum to include Nessie:
+Next, we need to update the Akri protocol handler enum to include http:
 
 ```rust
 pub enum ProtocolHandler {
-    nessie(NessieDiscoveryHandlerConfig),
+    http(HTTPDiscoveryHandlerConfig),
     ...
 }
 ```
 
-Finally, we need to add Nessie to the CRD yaml so that Kubernetes can properly validate any one attempting to configure Akri to search for Nessie.  To do this, we need to modify `deployment/helm/crds/akri-configuration-crd.yaml`:
-
-```yaml
-openAPIV3Schema:
-    type: object
-    properties:
-    spec:
-        type: object
-        properties:
-        protocol: # {{ProtocolHandler}}
-            type: object
-            properties:
-                nessie: # {{NessieDiscoveryHandler}} <--- add this line
-                    type: object                                # <--- add this line
-                    properties:                                 # <--- add this line
-                        nessieUrl:                              # <--- add this line
-                            type: string                        # <--- add this line...
-```
-
-### Create a sample protocol broker
-The final step, is to create a protocol broker that will make Nessie available to the cluster.  The broker can be written in any language as it will be deployed as an individual pod; however, for this example, we will make a Rust broker. We can use cargo to create our project by navigating to `samples/brokers` and running `cargo new nessie`.  Once the nessie project has been created, it can be added to the greater Akri project by adding `"samples/brokers/nessie"` to the **members** in `./Cargo.toml`.
-
-As a simple strategy, we can split the broker implementation into parts:
+Finally, we need to add http to the CRD yaml so that Kubernetes can properly validate any one attempting to configure Akri to search for HTTP devices.  To do this, we need to modify `deployment/helm/crds/akri-configuration-crd.yaml`:
 
-1. Create a shared buffer for the data
-1. Accessing the "nessie" data
-1. Exposing the "nessie" data to the cluster
+> **NOTE** Making this change means you must `helm install` a copy of this directory **not** deislabs/akri hosted

```

Comment:
Maybe add: "We will cover this when we deploy our solution" and link to https://github.com/deislabs/akri/blob/907ea8b365dcd8e207ba8ea9beeef6f6c9607db0/docs/extensibility.md#deploy-akri. It may not make immediate sense to folks without an example